### PR TITLE
Disrupting Nate Gatsby site submission

### PR DESCRIPTION
**Pull request by nathanolmstead**
Gatsby V2 site submission.

### CODE REVIEW

File: docs/sites.yml [hunk 1]
```diff
@@ -3154,3 +3154,16 @@
   built_by: Gabe Ragland
   built_by_url: https://twitter.com/gabe_ragland
   featured: false
+- title: Disrupting Nate
+  description: >
+    Ketogenic Diet, Podcasts, and Blockchain.
+  url: https://www.disruptingnate.com/
+  main_url: https://disruptingnate.com/
+  categories:
```
Comment (DSchau) [inline docs/sites.yml]: These categories need to be indented a level (thus the failing CircleCI lint job!)

File: docs/sites.yml [hunk 2]
```diff
@@ -3154,3 +3154,16 @@
...
+  - Technology
```
Comment (DSchau) [inline docs/sites.yml]: Can we keep these to categories we've seen before or that are more common? Maybe just Technology and Podcasts is a good start?

File: docs/sites.yml [hunk 3]
```diff
@@ -3154,3 +3154,16 @@
   built_by: Gabe Ragland
   built_by_url: https://twitter.com/gabe_ragland
   featured: false
+- title: Disrupting Nate
+  description: >
+    Ketogenic Diet, Podcasts, and Blockchain.
+  url: https://www.disruptingnate.com/
```
Comment (DSchau) [inline docs/sites.yml]: This appears to be down for me?

Comment (nathanolmstead) [reply docs/sites.yml]: Was deploying and still working out a DNS issue, need to type in www

File: docs/sites.yml (hunk 2 above)
Comment (nathanolmstead) [reply docs/sites.yml]: Updated

File: docs/sites.yml (hunk 1 above)
Comment (nathanolmstead) [reply docs/sites.yml]: Updated

**Comment by nathanolmstead** [comment]
Changes committed. Please review.

**Pull request merged**